src/associated-items.rst

```diff
@@ -91,6 +91,14 @@ within an :t:`implementation`.
 :dp:`fls_bx7931x4155h`
 An :t:`associated implementation type` shall have an :t:`initialization type`.
 
+:dp:`fls_bnTcCbDvdp94`
+An :t:`associated trait item` is an :t:`associated item` that appears
+within a :t:`trait`.
+
+:dp:`fls_N3cdn4lCZ2Bf`
+An :t:`associated trait implementation item` is an :t:`associated item` that
```

Comment:
Would it be clearer to write a "trait implementation associated item"? Since the associated item is associated to the trait implementation? Or at least a dash: trait-implementation associated item?
Or is it against the standard (and we would need to change it everywhere!)

Reply:
we already have things like `associated trait constant` and such, so that's where this order comes from

Reply:
I understand it as "trait constant" being the main thing, then pre-fixing it with "associated" to be more specific. "trait associated constant" would not flow as well.

Reply:
The 3 word terms are ok, it is mostly the 4 word terms that become unclear.